### DISCUSSION TITLE
Quasar

### DISCUSSION THREAD

**Discussion by priyanka-kurve** [quasarframework/quasar#12915] [General - Components / Directives / etc]
hi i want help in my code as i am new to quasar

i have data from api like
[
    {
        "id": 1,
        "name": "Site11",
        "metadata": {
            "key1": 1,
            "key2": 1,
            "key3": 1,
            "key9": 10
        }
    },
    {
        "id": 2,
        "name": "Site2",
        "metadata": {
            "key2": 2
        }
    }]

i am making q-table in which i want one column of keys and other of values but it is not populating

![Screenshot ](https://user-images.githubusercontent.com/60501475/159859717-3f7d56b9-b600-4464-b9e9-36f2c66e5c05.jpg)

![2](https://user-images.githubusercontent.com/60501475/159860912-75dee50d-dc66-40cc-8f1a-654cc08b778a.jpg)

 Columns: [
        {
          name: 'metadata',
          align: 'center',
          label: 'Key Name',
          field: row => row.properties.keys,
          sortable: true
        },
        {
          name: 'metadata',
          align: 'center',
          label: 'Key Value',
          field: row => row.properties.metadata.Value,
          sortable: true
        },
        

         {
          name: 'Action',
          required: true,
          label: 'Action',
          align: 'left',
          
        }
      ]

it is not populating data

**Comment by MilosPaunovic** [comment]
Seems like you table does not recieve proper data. Create a reproduction, so someone could help you easier.
Use this as a base: [https://codepen.io/milospaunovic/pen/WNdRQJa?editors=101](https://codepen.io/milospaunovic/pen/WNdRQJa?editors=101)
